src/app/api/postTerms.ts

```diff
@@ -0,0 +1,28 @@
+import { apiWrapper } from "./apiWrapper";
+
+interface TermsPayload {
+  address: string;
+  termsAccepted: boolean;
+  publicKey: string;
+}
+
+export const postTerms = async (
```

Comment:
```suggestion
export const postLogTermsAcceptance = async (
```

Let's be explicit here. We are not posting any terms, we are doing a request for them being logged in the back-end. Further a comment explaining what the function does and what it returns will be good for code quality.